export async function checkResults() {
    try {
        console.log('Checking test results...');
    } catch (error) {
        console.log(error);
    }
}